styles/MemberCard.module.css

```diff
@@ -46,7 +46,3 @@
 .card__title {
   font-weight: 500;
 }
```

Comment:
Do we need additional font-weight? Is there any pre-defined font style in our global CSS files?

Reply:
We do not need additional font-weight. The font-size & line-height are pre-defined in the global CSS.